cypress/integration/proposal_administration.ts

```diff
@@ -531,13 +533,13 @@ context('Proposal administration tests', () => {
 
       cy.finishedLoading();
 
-      cy.get('table tbody tr').eq(0).contains(proposalFixedName);
-      cy.contains('Title').dblclick();
       cy.get('table tbody tr').eq(1).contains(proposalFixedName);
+      cy.contains('Title').click();
```

Comment:
I couldn't figure out why the table is populated in a different order with this change (I'm not sure how the original results were ordered, but if someone knows I can always set a default sort) but sorting in the other order and reversing the assertions tests the same functionality.